test/Altinn.Notifications.Tests/Notifications.Core/TestingServices/EmailNotificationServiceTests.cs

```diff
@@ -136,6 +136,32 @@ public async Task CreateEmailNotification_ToAddressDefined_ResultFailedRecipient
         repoMock.Verify();
     }
 
+    [Fact]
+    public async Task UpdateSendStatus()
```

Comment:
I would like to see a slightly better test scenario description as the method name.